### DISCUSSION TITLE
Is there a way to create invite links to a group using this module?

### DISCUSSION THREAD

**Discussion by istoleabread** [eternnoir/pyTelegramBotAPI#1496] [Q&A]
I want to auto generate a group's invite link and send it to users joining another group, to prevent spam attacks and people who join without any reason. Is there a way I can do it?

**Comment by istoleabread** [comment]
Nvm, I got but I can't get the invite_link from ChatInviteLink class. Any help?

**Comment by Badiboy** [comment]
What means "I can't get"?

**Reply by istoleabread** [reply]
Nvm, I asked on the telegram and solved this.
Thanks alot!
